conntrack: Add operational metrics on errors for hash computation and aggregators

ronensc opened the pull request:
This PR adds the following operational metrics that count errors during hash computation and in the aggregators.
```
conntrack_aggregator_errors{error="MissingFieldError",field="Bytes"} 11
conntrack_aggregator_errors{error="Float64ConversionError",field="Bytes"} 22
conntrack_hash_errors{error="MissingFieldError",field="Missing"} 33
```

jpinsonneau reviewed (dismissed):
LGTM in terms of code (not tested). Thanks @ronensc !

ronensc commented:
Rebased and added another operational metric to count how connections are getting ended per scheduling group, either by getting `FIN` flag or by timing out.
```
conntrack_end_connections{group="0: DEFAULT",reason="FIN_flag"} 11
conntrack_end_connections{group="0: DEFAULT",reason="timeout"} 22
```

ronensc commented:
/approve
